fvm/transactionInvoker.go

```diff
@@ -158,7 +164,9 @@ func (i *TransactionInvoker) Process(
 	if feesError != nil {
 		txError = feesError
 	}
-	sth.EnableLimitEnforcement()
+	if !payerIsServiceAccount {
+		sth.EnableLimitEnforcement()
```

Comment:
there are more `EnableLimitEnforcement` calls in `AddEncodedAccountKey` and `CreateAccount` if those would be called during a transaction, then the limit would be reenabled...

It might be better to put `payerIsServiceAccount` inside the `StateHolder` and use it in `EnforceInteractionLimits`.

Would it make sense to increase the limits instead of disabling them?

Is this meant as a very temporary fix? If that is the case then it is fine, as the transactions that are causing problems are not calling  `AddEncodedAccountKey` or `CreateAccount`.

Reply:
good idea, fixed it.